pkg/lib/size/size_test.go

```diff
@@ -0,0 +1,55 @@
+package size
+
+import (
+	"testing"
+)
+
+func TestSize(t *testing.T) {
+	type args struct {
+		endpoint  string
+		accessKey string
+		secKey	  string
+	}
+	tests := []struct {
+		name  string
+		args  args
+		low   int64 // lowest acceptable size due to variations in iso hdrs
+		exact int64
+		high  int64 // highest acceptable size due to variations in iso hdrs
+	}{
+		{
+			name:  "small tinyCore.iso file size",
+			args:  args{"file:///root/go/src/kubevirt.io/containerized-data-importer/tinyCore.iso", "", ""},
```

Comment:
these paths should be set dynamically using the `os` package and made relative to the test file rather than absolute.

Reply:
e.g.
```
	srcIso, err := filepath.Abs("../../../test/images/tinycore.iso")
	if err != nil {
		t.Fatalf("Failed to get source image path: %s\n", err)
	}
```

Reply:
agree. I forgot to delete some debugging test case.